src/components/ui/Navigation.styled.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const NavBottom = styled.article`
+	max-width: 800px;
```

Comment:
Is there any particular reason for this "magic number"? :)

Reply:
Hey CJ, there is actually no reason at all for this restriction, so actually something like magic . I have copied all styles so far from previous projects together and roughly adapted them to the project. I sincerely hope to find more time for my styling by the weekend, but i will definitely correct the mistakes you marked. The color scheme will probably have to wait until the weekend :-D